<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="B300LCOK" ignored="0" name="Default test" tests="2" failures="1" timestamp="2023-02-28T00:17:30 IST" time="350.773" errors="0">
  <testcase name="runScenario" time="63.608" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="284.178" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="Object healing  unsuccesful,please refer self healing report.">
      <![CDATA[java.lang.AssertionError: Object healing  unsuccesful,please refer self healing report.
at org.testng.Assert.fail(Assert.java:97)
at com.scripted.web.WebHandlers.initHeal(WebHandlers.java:138)
at com.scripted.web.WebHandlers.clickByJsExecutor(WebHandlers.java:191)
at com.scripted.web.webstepdefs.NewPage.select_shop(NewPage.java:143)
at com.scripted.web.webstepdefs.stepdefinitions.select_pro_add_cart(stepdefinitions.java:190)
at âœ½.user select the product and add to the cart(file:Features/selfhealing/saveMart.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
